src/qinfer/resamplers.py

```diff
@@ -61,7 +62,7 @@
 
 class Resampler(with_metaclass(ABCMeta, object)):
     @abstractmethod
-    def __call__(self,  model, particle_weights, particle_locations,
+    def __call__(self,  model, particle_dist,
```

Comment:
I like this change too, and it needs to happen some time. The number of custom qinfer resamplers out there that we don't know about is probably zero :P . Coincidentally, I wrote one yesterday.